doc/source/management/innodb_kill_idle_trx.rst

```diff
@@ -1,28 +1,52 @@
 .. _innodb_kill_idle_trx:
 
-========================
- Kill Idle Transactions
-========================
+======================
+Kill Idle Transactions
+======================
 
-This feature limits the age of idle |XtraDB| transactions. If a transaction is idle for more seconds than the threshold specified, it will be killed. This prevents users from blocking purge by mistake.
+This feature limits the age of idle transactions. If a transaction is idle for
```

Comment:
"of idle transactions, for all transactional storage engines." (to avoid confusion, as specifically InnoDB purge is mentioned below)